bookkeeper-server/pom.xml

```diff
@@ -217,11 +217,21 @@
       <version>${netty.version}</version>
     </dependency>
     <dependency>
-        <groupId>org.apache.hadoop</groupId>
-        <artifactId>hadoop-minikdc</artifactId>
-        <version>2.7.3</version>
-        <scope>test</scope>
-    </dependency>          
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-minikdc</artifactId>
+      <version>2.7.3</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
```

Comment:
please consider using http://json-b.net/ which is the current (brand new) standard for Java-JSON mapping.
org.json is really bad.
otherwise we could  use Jackson Mapper, which is actually the best